tools/functional-tester/etcd-tester/election_runner_stresser.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2016 The etcd Authors
```

Comment:
There are three new _stresser.go files that basically do the same thing; this is way too much code duplication to be properly maintained. Please abstract it.